README.md

```diff
@@ -2,7 +2,7 @@
 
 konk - Kubernetes On Kubernetes - is a tool for deploying an independent Kubernetes API server within Kubernetes.
 
-konk can be used as part of a larger application to manage resources via CustomResourceDefinitions and implement a CRUD API for those resources by leveraging kube-apiserver.
+konk can be used as part of a larger application to manage resources via CustomResourceDefinitions and implement a CRUD API for those resources by leveraging kube-apiserver. Or implement an [extension API server](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) without worrying about [breaking the parent cluster with a non-compliant API](https://github.com/kubernetes/kubernetes/issues/96066).
```

Comment:
Should we point them to a toolkit instead like https://github.com/kubernetes-sigs/apiserver-builder-alpha. From there, it links to other k8s docs about apiserver aggregation. There's a good description of CRD vs API Server.

We should remove all mentions of CustomResourceDefinitions here. Applications working with Konk need to bring their own storage layer. With CRD, the storage backend would be Konk's etcd which has no backup/fault tolerance gurantees